dali/base/dasds.cpp

```diff
@@ -5760,45 +5761,81 @@ class CStoreHelper : implements IStoreHelper, public CInterface
             constexpr const char *temporaryXmlFileSaveName = "dali_store_tmp.xml";
             OwnedIFileIO iXmlFileIOTmpStore = createUniqueFile(location, temporaryXmlFileSaveName, NULL, tmpStoreName);
             OwnedIFile iXmlFileTmpStore = createIFile(tmpStoreName);
```

Comment:
I think these can be commoned up and move into saveStoreFile, if it returns the IFile